charts/cosigned/values.yaml

```diff
@@ -0,0 +1,51 @@
+cosign:
+  # add the values in base64 encoded
+  cosignKey:
+  cosignPub:
+  cosignPassword:
+
+webhook:
+  name: webhook
+  image:
+    repository: gcr.io/projectsigstore/cosigned
+    # crane digest gcr.io/projectsigstore/cosigned:v1.2.0
+    version: sha256:55640ca4ad7803d0002d97c0200636b37318570702a74bd1f7650e1fda6206ae
```

Comment:
To preserve the image tag information the version could be changed to `v1.2.0@sha256:55640ca4ad7803d0002d97c0200636b37318570702a74bd1f7650e1fda6206ae`.

The format `name[:tag|@digest]` is supported by k8s. This admission webhook tool performs this action on pods => https://github.com/google/k8s-digester

Docker and podman also support this notion:
- https://docs.docker.com/engine/reference/commandline/image_pull/
- https://docs.podman.io/en/latest/markdown/podman-pull.1.html

Reply:
agree with you, but we can change that in a follow-up, I did in this way per Andrew's request to match the other chart we have here, but I agree to have the tag on it that will be easier :)